docs/release/changelog.md

```diff
@@ -0,0 +1,71 @@
+# Generate the changelog for a release
```

Comment:
Would you mind linking here from the relevant step of `docs/development/release.md`?
(Probably, we should also also move that doc into the `docs/release` into this directory.)

Reply:
Never mind, I see you did that in https://github.com/ray-project/kuberay/pull/834/files.
Thanks!